serverless/cmd/distribute/github/main.go

```diff
@@ -164,6 +173,33 @@ func distributeOnce(ctx context.Context, opts *options, repo github.Repository)
 	return repo.CreatePR(ctx, fmt.Sprintf("Witness %s@%d", opts.witSigV.Name(), wCp.Size), branchName)
 }
 
+func alreadyPresent(cpBody string, logDir string, repo github.Repository, wSigV note.Verifier) (bool, error) {
```

Comment:
Maybe alreadyIntegrated would be clearer? This isn't just checking that we have generated this checkpoint before, but that it's been merged into the distributor.

Reply:
done